bodhi-server/pyproject.toml

```diff
@@ -1,3 +1,167 @@
+# Possible options are at https://pypi.python.org/pypi?%3Aaction=list_classifiers
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: System :: Software Distribution",
+    "Framework :: Pyramid",
+    "Programming Language :: JavaScript",
+    "Topic :: Internet :: WWW/HTTP",
+    "Topic :: Internet :: WWW/HTTP :: WSGI :: Application",
+]
+packages = [
+    { include = "bodhi" },
+    { include = "tests", format = "sdist"}
+]
+include = [
+    { path = "*.txt", format = "sdist" },
+    { path = "*.ini", format = "sdist" },
+    { path = "*.cfg", format = "sdist" },
+    { path = "*.rst", format = "sdist" },
+    { path = "celeryconfig.py", format = "sdist" },
+    { path = "COPYING", format = "sdist"},
+    { path = "bodhi/server/locale/.placeholder", format = "sdist" },
+    { path = "bodhi/*.ico", format = "sdist" },
+    { path = "bodhi/*.png", format = "sdist" },
+    { path = "bodhi/*.css", format = "sdist" },
+    { path = "bodhi/*.gif", format = "sdist" },
+    { path = "bodhi/*.jpg", format = "sdist" },
+    { path = "bodhi/*.pt", format = "sdist" },
+    { path = "bodhi/*.txt", format = "sdist" },
+    { path = "bodhi/*.mak", format = "sdist" },
+    { path = "bodhi/*.mako", format = "sdist" },
+    { path = "bodhi/*.js", format = "sdist" },
+    { path = "bodhi/*.html", format = "sdist" },
+    { path = "bodhi/*.xml", format = "sdist" },
+    { path = "bodhi/*.ttf", format = "sdist" },
+    { path = "bodhi/*.woff", format = "sdist" },
+    { path = "bodhi/*.woff2", format = "sdist" },
+    { path = "bodhi/*.eot", format = "sdist" },
+    { path = "bodhi/*.otf", format = "sdist" },
+    { path = "bodhi/*.svg", format = "sdist" },
+    { path = "bodhi/*.tpl", format = "sdist" },
+    { path = "bodhi/*.rst", format = "sdist" },
+    { path ="apache/*", format = "sdist" },
+    { path = "docs", format = "sdist" },
+        ]
+exclude = ["development.ini", "docs/_build"]
+
+#paster_plugins = pyramid
+#message_extractors = 
+#    . = []
+
+
+[tool.poetry.dev-dependencies]
+bodhi-messages = {path = "../bodhi-messages"}
+pytest = ">=6.2.2"
+
+[tool.poetry.dependencies]
+authlib = ">=0.15.3"
+Jinja2 = ">=2.11.3"
+Markdown = "^3.3.6"
+PyYAML = ">=5.4.1"
+SQLAlchemy = "^1.3.24"
+alembic = "^1.5.5"
+arrow = ">=0.17.0"
+backoff = "^1.10.0"
+bleach = ">=3.2.3"
+bodhi-messages = "^5.7.4"
+celery = [
+    {version = "<=5.2.2, >=5.0.5", python = "3.9"},
+    {version = "<=5.2.3, >=5.0.5", python = "3.10"}
+]
+click = ">=7.1.2"
+colander = "^1.8.3"
+cornice = "^5.0.3"
+"dogpile.cache" = "^1.1.2"
+fedora-messaging = "^3.0.0"
+feedgen= "^0.9.0"
+koji = "^1.27.1"
+#libcomps ="^0.1.18"
+libcomps ="*"
```

Comment:
Could you explain this in a comment please?

Reply:
I probably wanted to see which libcomps would be installed.
You mean to remove the wildcard or leave it and comment which problem we were solving with it?

Reply:
Well, if we actually need the wildcard, then we need a comment to explain why it's needed. If we don't need it, then it should be removed :-)